client/electron/go_vpn_tunnel.ts

```diff
@@ -67,8 +67,10 @@ export class GoVpnTunnel implements VpnTunnel {
     private readonly routing: RoutingDaemon,
     readonly transportConfig: TransportConfigJson
   ) {
-    this.tun2socks = new GoTun2socks(transportConfig);
-    this.connectivityChecker = new GoTun2socks(transportConfig);
+    // We need two GoTun2socks instances because we need to update the UDP connectivity
+    // using this.connectivityChecker while this.tun2socks is still running.
+    this.tun2socks = new GoTun2socks();
+    this.connectivityChecker = new GoTun2socks();
```

Comment:
Don't use GoTun2socks. just call the standalone connectivity function I mention below.

Reply:
Done